api/search/__init__.py

```diff
@@ -35,6 +35,20 @@ def es_query(input_query, doc_type, min_score=0.5, additional_filter=None, sourc
 
     return query
 
+def es_aggs(doc_type, field_name):
+    query = {
+      "size": 0,
+      "aggs": {}
+    }
+
+    query['aggs'][field_name] = {
+        "terms": {
+            "field": '.'.join((doc_type, field_name, 'exact_'+field_name))
```

Comment:
I assume this is the right way of using `field_name` for aggregation. 
It is like the `exact_` field is a subfield, right?